alembic/versions/5ab82f23e15c_add_shift.py

```diff
@@ -0,0 +1,53 @@
+"""add shift
+
+Revision ID: 5ab82f23e15c
+Revises: 82f5a89fbe59
+Create Date: 2022-02-01 05:24:23.927168
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5ab82f23e15c'
+down_revision = '82f5a89fbe59'
```

Comment:
will need to update the down revision given that there are new migrations.

Reply:
The migration script is now complaining about bandpasses... although I expect this must be not related to this PR